src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days === 1 || days === 2) {
```

Comment:
Use <> operators instead of enumerating all possible options
```suggestion
  if (days < 3) {
```